docs/reference/modules/asset.md

```diff
@@ -0,0 +1,205 @@
+```
+</AposCodeBlock>
+
+### `clear-cache`
+Some changes to the webpack configuration will not be detected automatically and will cause incorrect output. While this is rare, the caches for webpack can be emptied from the command line using the `clear-cache` task.
+
+#### Example
+
+<AposCodeBlock>
+
+```sh
+node app @apostrophecms/asset:clear-cache
+```
+</AposCodeBlock>
+
+## Webpack configuration
+
+The webpack configuration can be extended from any module. The `asset` module exposes three different properties for accomplishing this.
+
+## Webpack options
+|  Property | Type | Description |
+|---|---|---|
+| [`extensions`](#extensions) | Object | Adds extra processing functionality to webpack. |
+| [`extensionOptions`](#extensionoptions) | Object | Adds functionality to any extensions added through the `extensions` option. |
+| [`bundles`](#bundles) | Object | |
+
+### `extensions`
+The `extensions` option allows the addition of custom methods for processing the Javascript and SCSS in the `ui/src` folder. The schema used within the option conforms to that used for configuring webpack, which you can learn more about [here](https://webpack.js.org/configuration/). This property can take either a simple object with a name property or a function that returns an object. 
+
+#### `extensions` simple object
+
+#### Example
+
+Modification of the `resolve` property of the webpack configuration using a simple object. New values will be merged with existing values:
+
+<AposCodeBlock>
+
+```javascript
+const path = require("path");
+
+module.exports = {
+ // ...
+ webpack: {
+   extensions: {
+     modifyResolve: {
+       resolve: {
+         extensions: [ '.jsx' ],
+         alias: {
+           'Utilities': path.join(process.cwd(), 'lib/utils')
+         }
+       }
+     }
+   }
+ },
+ // ...
+};
+
+```
+<template v-slot:caption>
+modules/my-module/index.js
+</template>
+</AposCodeBlock>
+
+::: note
+Make sure the new values are of the same type as the existing values, or they will replace the existing values, e.g., the webpack configuration `extensions` property takes an array, whereas `alias` takes an object.
+:::
+
+This will add both processing for files with an extension of `.jsx` and expose a shortcut to the `lib/utils` folder that we can use in our frontend files. For example, to import a custom method from our `methods.js` library in this folder.
+
+<AposCodeBlock>
+
+```javascript
+import  { customMethod } from ('Utilities/methods.js');
+
+export default () => {
+// Code utilizing customMethod
+};
+```
+<template v-slot:caption>
+modules/my-module/ui/src/index.js
+</template>
+</AposCodeBlock>
+
+Adding a unique name for the object within the apostrophe `extensions` property is important. Passing an object with the same name to an `extensions` property of another module will cause a conflict. The configuration file will be set to the `extensions` object in the last loaded module.
```

Comment:
a unique name for the extension

If an extension of the same name is added by two modules, the last module configured wins. This is intentional, it is a feature to allow overrides. So it doesn't "cause a conflict," but it does mean you need to name them well.

Reply:
The last sentence here doesn't quite make sense.

Reply:
I changed the structure of this paragraph to reflect your comments.